File: Python/TestLambda.py
import json
import time


def lambda_handler(event, context):

    current_milli_time = int(round(time.time() * 1000))
    api_epoch_time = int(event['requestContext']['requestTimeEpoch'])

    rs_body = f'Python Test Lambda took {current_milli_time - api_epoch_time}ms to start'
    print(rs_body);

    return {
        'statusCode': 200,
        'body': rs_body
    }

File: Golang/TestLambda.go
package main

import (
	"context"
	"fmt"
	"time"
	"strconv"
    "github.com/aws/aws-lambda-go/events"
	"github.com/aws/aws-lambda-go/lambda"
)

func HandleRequest(ctx context.Context, request events.APIGatewayWebsocketProxyRequest) (events.APIGatewayProxyResponse, error) {

    currentTime := int64(time.Now().UnixNano() / int64(time.Millisecond))
    apiEpochTime := request.RequestContext.RequestTimeEpoch
    rsBody := "Go Test Lambda took " + strconv.FormatInt((currentTime - apiEpochTime), 10) + "ms to start";

    fmt.Println(rsBody)
    return events.APIGatewayProxyResponse{Body: rsBody, StatusCode: 200}, nil
}

func main() {
    lambda.Start(HandleRequest)
}
